ai_adapter.py

```
@@ -89,62 +91,62 @@ async def query_chain(message):
         logger.info("Invoking expert chain...")
-        answer = invoke_model(messages)
+        result = invoke_model(messages)
         json_result = {}
-        logger.info(f"Expert chain invoked. Result is `{answer}`")
+        logger.info(f"Expert chain invoked. Result is `{result}`")
         try:
             # try to parse a valid JSON response from the main expert engine
-            json_result = json.loads(answer)
+            json_result = json.loads(result)
             logger.info("Engine chain returned valid JSON.")
         except:
```

Ruff

Do not use bare `except`

(E722)